common/core/nesteddisplay.cpp

```diff
@@ -207,4 +440,47 @@ bool NestedDisplay::GetDisplayName(uint32_t *size, char *name) {
   return true;
 }
 
+#ifdef NESTED_DISPLAY_SUPPORT
+int NestedDisplay::start_sock_service() {
+  struct sockaddr_in server_addr;
+  msock_fd = socket(AF_INET, SOCK_STREAM, 0);
+  if (msock_fd < 0) {
+    ETRACE("Cannot create socket fd:%d", msock_fd);
+    return -1;
+  }
+
+  memset(&server_addr, 0, sizeof(server_addr));
+  int portno = port_;
+  server_addr.sin_family = AF_INET;
+  server_addr.sin_addr.s_addr = INADDR_ANY;
+  server_addr.sin_port = htons(portno);
+
+  if (bind(msock_fd, (struct sockaddr *)&server_addr, sizeof(server_addr)) <
+      0) {
+    close(msock_fd);
+    msock_fd = -1;
+    return -1;
+  }
+
+  listen(msock_fd, 1);
+  signal(SIGPIPE, signal_callback_handler);
+  return 0;
+}
+
+int NestedDisplay::hyper_communication_network_send_data(void *data, int len) {
```

Comment:
Rename this to HyperCommunicationNetworkSendData

Reply:
ok, will follow it.